modules/mod_signal/mod_signal.erl

```diff
@@ -182,11 +195,11 @@ handle_call({'connect', Signal, Slot}, _From, State) ->
     TagSet = ets:lookup(State#state.slots, SignalType),
 
     Objects = case lists:member({tags, SignalType, SignalTags}, TagSet) of
-                  true ->
-                      {slot, Key, Slot};
-                  false ->
-                      [{slot, Key, Slot}, {tags, SignalType, SignalTags}]
-              end,
+        true ->
+            {slot, Key, Slot};
+        false ->
+            [{slot, Key, Slot}, {tags, SignalType, SignalTags}]
+    end,
```

Comment:
Actually I find these *left-aligned* case statements uglier and much less readable then the normal *aligned below the case* versions.